src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -857,6 +857,12 @@ void VM_Version::get_processor_features() {
   }
 #endif
 
+  // Check if processor has Intel Ecore
+  if (FLAG_IS_DEFAULT(DoEcoreOpt) && is_intel() && cpu_family() == 6 &&
+    (_model == 0x97 || _model == 0xAC || _model == 0xAF)) {
```

Comment:
Correct, CPUID is CPU specific not core specific, so settings should apply to both.

Reply:
As per JVM specification section 2.8 "The floating-point instructions of the Java Virtual Machine do not throw
exceptions, trap, or otherwise signal the IEEE 754 exceptional conditions of
invalid operation, division by zero, overflow, underflow, or inexact.", thus JVM does not check these exceptions.

Your patch is always setting lower 6 bits of MXCSR on hybrid CPUs which has both E and P core, do you see any concerns if these bits are default ON for other server targets.